docs: serverless deploy feedback (#86357)
vercel/next.js: docs/01-app/03-api-reference/01-directives/use-cache.mdx

| Environment     | Runtime Caching Behavior                                                                                                                                          |
| --------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------- |

> | **Serverless**  | Cache entries typically not persist across requests (each request can be a different instance). Build-time caching works normally.                                |

Reviewer: sorry just stumbled on this pr in slack and was interested but i see a small typo

"typically don't" or just "typically", instead of "typically not"

Author: oj haha - editing directly in the GitHub UI has its quirks - ty

> | **Serverless**  | Cache entries typically do not persist across requests (each request can be a different instance). Build-time caching works normally.                             |

Reviewer 2: ```suggestion
| **Serverless**  | Cache entries typically don't persist across requests (each request can be a different instance). Build-time caching works normally.                             |
```
